inc/class-theme.php

```diff
@@ -1476,4 +1476,25 @@ public function setAppEndpoint($appEndpoint)
 
         return $this;
     }
+
+        /**
+     * Get the value of showClassicSwitch
+     */ 
+    public function getShowClassicSwitch()
+    {
+        return $this->showClassicSwitch;
+    }
+
+    /**
+     * Set the value of showClassicSwitch
+     *
+     * @return  self
+     */ 
+    public function setShowClassicSwitch($showClassicSwitch)
+    {
+        $this->showClassicSwitch = $showClassicSwitch;
+
+        return $this;
```

Comment:
Why do you need to return $this in this setter method? What is the context in which it is being called?